src/normalizer.ts

```diff
@@ -222,6 +223,29 @@ rules.set('Transform const to singleton enum', schema => {
   }
 })
 
+rules.set('Transform nullable to null type', schema => {
+  if (schema.nullable !== true) {
+    return
+  }
+
+  delete schema.nullable
```

Comment:
Do we need to delete the `nullable` property if it has any other value than `true`? e.g. if it is `false`, I don't imagine we'd actually want to put effort into searching for `null` in enum/const or `"null"` in type/anyOf/oneOf, but we might still want to delete the property? Or is that unnecessary?